src/components/Shared/Navbar/SignedUser.tsx

```diff
@@ -36,6 +37,7 @@ const SignedUser: FC = () => {
   const { allowed: staffMode } = useStaffMode();
   const { theme, setTheme } = useTheme();
   const { disconnect } = useDisconnect();
+  const { refetch } = useQuery(RecommendedProfilesDocument);
```

Comment:
We dont want to refetch this 🙇🏼 cuz we removed recommendation for logged out users 🙏🏼

Reply:
Oh okey